tests/jballerina-unit-test/src/test/resources/test-src/query/string-query-expression.bal

```diff
@@ -336,3 +400,25 @@ function testQueryExprWithLimitForStringResultV2() returns string {
 
    return outputNameString;
 }
+
+function assertEquality(any|error actual, any|error expected) {
+    if expected is anydata && actual is anydata && expected == actual {
+        return;
+    }
+
+    if expected === actual {
+        return;
+    }
```

Comment:
Better if the brackets are added here.

```suggestion
    if (expected is anydata && actual is anydata && expected == actual) {
        return;
    }

    if (expected === actual) {
        return;
    }
```

Reply:
I guess the best practice is to not add brackets in ballerina. @pcnfernando @LakshanWeerasinghe wdyt?